pkg/networkpolicy/controller.go

```diff
@@ -123,7 +123,7 @@ func newController(client clientset.Interface,
 		nft:    nft,
 		queue: workqueue.NewTypedRateLimitingQueueWithConfig(
 			workqueue.DefaultTypedControllerRateLimiter[string](),
-			workqueue.TypedRateLimitingQueueConfig[string]{Name: "controllerName"},
+			workqueue.TypedRateLimitingQueueConfig[string]{Name: controllerName},
```

Comment:
1. lol
2. commit message ("bump dependencies") does not match the commit body?

Reply:
1 is something we have in all kubernetes components :) , the os.Args does not give the passed flags. it is very useful for debugging

2. now I understand why I have a merge conflict, it seems I have an old version, and once I rebased over the main  I got the dependency update :)